bin/grant-server/main.go

```diff
@@ -114,6 +114,19 @@ func setupRouter(ctx context.Context, logger *logrus.Logger) (context.Context, *
 		r.Mount("/v2/attestations/safetynet", proxyRouter)
 	}
 
+	go func() {
```

Comment:
since this isn't really related to the router I would probably put it in `main()`

Reply:
so, i thought about that, but didn't think it was right, i guess is the only word i can think of, since i would have to send out the promotion service too.